core-bundle/src/Resources/contao/library/Contao/Model.php

```diff
@@ -583,7 +583,8 @@ public function getRelated($strKey, array $arrOptions=array())
 		// The relation does not exist
 		if (!isset($this->arrRelations[$strKey]))
 		{
-			throw new \Exception("Field $strKey does not seem to be related");
+			$table = static::getTable();
```

Comment:
CS needs a empty line after `$table = static::getTable();`

Reply:
Thanks, added.

Any reason why this doesn't get fixed or flagged by:

````
tools/ecs/vendor/bin/ecs check core-bundle/src/Resources/contao/library/Contao/Model.php --config tools/ecs/vendor/contao/easy-coding-standard/config/default.php --no-progress-bar --ansi
````

As I understand, this is the same check that is run by the workflow. 
Is there any different configuration?

Reply:
did you miss the `--fix` flag?

Reply:
I was running ecs with the default configuration:

````
--config tools/ecs/vendor/contao/easy-coding-standard/config/default.php
````

but this skips the legacy code in `Resources/contao/`, which has to be checked with the legacy configuration:

````
--config tools/ecs/vendor/contao/easy-coding-standard/config/legacy.php
````

My mistake.

Reply:
👍